src/main/java/frc/robot/subsystems/mailbox/Mailbox.java

```diff
@@ -11,12 +11,30 @@
 
 package frc.robot.subsystems.mailbox;
 
+import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
 
 /** Subsystem for the mailbox that outputs game pieces from our robot. */
 public class Mailbox extends SubsystemBase {
+
+  /** Limit switch that detects when the mailbox is raised. */
+  private DigitalInput limitSwitch;
+
   /** Creates a new Mailbox. */
-  public Mailbox() {}
+  public Mailbox() {
+
+    this.limitSwitch = new DigitalInput(Constants.Mailbox.MAILBOX_LIMIT_SWITCH_DIO_PORT);
+  }
+
+  /**
+   * Gets the Mailbox Limit Switch's Value.
+   *
+   * @return true if the mailbox is fully raised. False otherwise.
+   */
+  public boolean getLimitSwitch() {
+    return limitSwitch.get();
```

Comment:
Prolly add a comment just above here that says that this will only work correctly if the limit switch is wired to be normally open

Reply:
Add it as a comment, not additional stuff in JavaDoc. Since it's an implementation note and not an API note, it doesn't belong in JavaDoc.

Also, I would phrase it as "assumes the limit switch is wired to be normally open"

Reply:
oh also I would put the comment in the actual method and not above the method header